### PR TITLE
chore: make clippy warnings into failures

### PR DESCRIPTION
**Pull request by sighphyre**

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
LGTM

File: server/src/http/unleash_client.rs [hunk 1]
```diff
@@ -188,7 +188,7 @@ impl UnleashClient {
         Self {
             urls: UnleashUrls::from_base_url(server_url),
             backing_client: new_reqwest_client(
-                "unleash_edge".into(),
+                "unleash_ede".into(),
```
Comment (chriswk) [inline server/src/http/unleash_client.rs:191]: Not the place to make a change just to have tarpaulin run.

Comment (sighphyre) [reply server/src/http/unleash_client.rs:191]: I promise I'm not going to merge this!

**Review by chriswk — APPROVED** [review]
Sure

**Pull request merged**